src/queries/organization.ts

```diff
@@ -13,12 +14,22 @@ export const useLatestElections = (limit = 5) => {
   })
 }
 
-export const usePaginatedElections = (page: number) => {
-  const { client, account } = useClient()
-
-  return useQuery({
-    enabled: !!account?.address,
-    queryKey: ['organization', 'elections', account?.address, page],
-    queryFn: async () => client.fetchElections({ organizationId: account?.address, page }),
-  })
+type PaginatedElectionsParams = {
+  page?: number
+  status?: FetchElectionsParameters['status']
 }
+
+export const paginatedElectionsQuery = (
+  account: AccountData,
+  client: VocdoniSDKClient,
+  params: PaginatedElectionsParams
+) => ({
+  enabled: !!account?.address,
+  queryKey: ['organization', 'elections', account?.address, params],
+  queryFn: async () =>
+    client.fetchElections({
+      organizationId: account?.address,
+      page: params.page ? Number(params.page) - 1 : 0,
+      status: params.status?.toUpperCase() as FetchElectionsParameters['status'],
+    }),
```

Comment:
Could be a good idea to type this return type using the same types as react query use:

```ts
): Pick<UseQueryOptions<ElectionListWithPagination>, 'enabled' | 'queryKey' | 'queryFn'>  =>({
```

Is not really needed, but just to ensure the types are correct. On this way, you ensure that the `queryClient.getQueryData` infers properly the return type also

Reply:
I'm not really sure why would I need to extra type stuff, when the type is properly inferred from the SDK
![imatge](https://github.com/user-attachments/assets/4095b637-bb5f-477b-a06d-a1788c9af6b9)